ios/Barometer.m

```diff
@@ -71,7 +66,7 @@ - (void) isAvailableWithResolver:(RCTPromiseResolveBlock) resolve
         
         if (altitudeData) {
             [self.bridge.eventDispatcher sendDeviceEventWithName:@"Barometer" body:@{
-                @"pressure" : altitudeData.pressure
+                @"pressure" : @(altitudeData.pressure.doubleValue * 10.0)
```

Comment:
Why is there a `* 10.0`?

Reply:
This is to be consistent with Android. Android returns hecto-Pascals, while iOS returns kilo-Pascals.

Reply:
Ah, gotcha 👍